src/api/admin.ts

```diff
@@ -122,7 +127,7 @@ app.patch('/providerKeys/:id', (req, res) => {
 })
 
 app.get('/providers', async (_, res) => {
-  const data = await Promise.all([goDaddy.getDomains()])
+  const data = await Promise.all([providers.dns_gd.getDomains()])
```

Comment:
I got a TS warning `Property 'getDomains' does not exist on type 'Function'.ts(2339)`
I don't know how to solve it